infrastructure/main.tf

```diff
@@ -1,8 +1,9 @@
+provider "azurerm" {}
 
-module "send-letter-producer-service-api" {
-  source   = "git@github.com/hmcts/terraform-module-webapp.git"
-  product  = "${var.product}"
-  location = "${var.location_api}"
+module "send-letter-producer-service" {
+  source   = "git@github.com:contino/moj-module-webapp?ref=master"
```

Comment:
can we update hmcts repo instead?

Reply:
i wonder who even created that repo. it's still empty

Reply:
😱